niap-cc/Permissions/Tester/app/src/main/java/com/android/certifications/niap/permissions/SignaturePermissionTester.java

```diff
@@ -1925,9 +1970,17 @@ public IBinder asBinder() {
 
         mPermissionTasks.put(permission.UPDATE_DEVICE_STATS,
                 new PermissionTest(false, () -> {
-                    mTransacts.invokeTransact(Transacts.BATTERY_STATS_SERVICE,
-                            Transacts.BATTERY_STATS_DESCRIPTOR,
-                            Transacts.noteStartAudio, 0, "test", mUid);
+
+                    if (mDeviceApiLevel == Build.VERSION_CODES.TIRAMISU) {
```

Comment:
Does this check need to be `>=`? Or is this transact only intended to be run on Android T and all newer platform releases can use the previous transact below?

Reply:
Thanks, Will fix it.

Reply:
Change looks good, thanks!